samples/BenchmarkDotNet.Samples/Program.cs

```diff
@@ -1,9 +1,75 @@
-﻿using BenchmarkDotNet.Running;
+﻿using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.InProcess.Emit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace BenchmarkDotNet.Samples
+namespace BenchmarkDotNet.Samples;
+
+public class Program
 {
-    public class Program
+    public static int Main(string[] args)
     {
-        public static void Main(string[] args) => BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
+#if DEBUG
+        ConsoleLogger.Default.WriteLineWarning("Benchmark is executed with DEBUG configuration.");
+        ConsoleLogger.Default.WriteLine();
+#endif
+
+        if (args.Length != 0)
+        {
+            ConsoleLogger.Default.WriteLine($"Start benchmarks with args: {string.Join(" ", args)}");
+            ConsoleLogger.Default.WriteLine();
+        }
+
+        IConfig config;
+#if DEBUG
+        config = GetDebugConfig();
+#else
+        config = null; // `DefaultConfig.Instance` is used.
+#endif
+
+        var summaries = BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly)
+                                         .Run(args, config)
+                                         .ToArray();
+
+        if (summaries.HasError())
+            return 1;
+
+        return 0;
+    }
+
+    private static ManualConfig GetDebugConfig()
+    {
+        return DefaultConfig.Instance
```

Comment:
I sometimes want to debug the separate process, this should take an option and choose `DebugBuildConfig` or `DebugInProcessConfig`.